frontend/src/components/atoms/Connections.tsx

```diff
@@ -1,52 +1,53 @@
-import { Button, Text } from '@chakra-ui/react';
+import { Button, Text, Wrap } from '@chakra-ui/react';
 import React from 'react';
-import { useDispatch } from 'react-redux';
-import { selectNode } from '../../state/reducers/ontologyReducer';
+import { correlationToColor } from '../../common/node';
+// import { useDispatch } from 'react-redux';
 import { Node } from '../../types/ontologyTypes';
 
 type ConnectionsProps = {
   connections: Array<Node>;
   titles: Array<string>;
+  color: string;
+  onClick: (selectedConnection: Node) => void;
```

Comment:
```suggestion
  handleOnClick: (selectedConnection: Node) => void;
```
I think having it named `handleOnClick` is cleaner.